File: output/CodeGen/avatar/Java/Python/atcoder_AGC012_A.py
class atcoder_AGC012_A:
    def main(self, args):
        sc = Scanner(System.in)
        n = sc.nextInt()
        array = [sc.nextInt() for _ in range(3 * n)]
        array.sort()
        head = len(array) - 2
        res = 0
        for i in range(n):
            res += array[head]
            head -= 2
        print(res)

File: output/StarCoder/avatar/Python/Java/atcoder_AGC012_A.java
import java.util.*;
import java.lang.*;
import java.io.*;
class atcoder_AGC012_A {
    public static void main ( String[] args ) {
        Scanner sc = new Scanner ( System.in );
        int n = sc.nextInt ( );
        int[] A = new int[n];
        for ( int i = 0 ; i < n ; i++ ) {
            A[i] = sc.nextInt ( );
        }
        Arrays.sort ( A );
        int ans = 0;
        for ( int i = A.length - 2 ; i >= A.length - 2 * n - 1 ; i -= 2 ) {
            ans += A[i];
        }
        System.out.println ( ans );
    }
}
